forem/forem
# Feature:Move Metric Rake Tasks to Sidekiq Cron

## What type of PR is this? (check all applicable)

## Description
Followup to #9789. This PR moves all metric jobs to Sidekiq Cron. Here is what the dashboard looks like after the jobs are present. 

![Screen Shot 2020-08-14 at 3 18 51 PM](https://user-images.githubusercontent.com/1813380/90289447-7c6d4d80-de41-11ea-9c62-cef16f90a969.png)

Jobs listed in Heroku so you can check cron times
![Screen Shot 2020-08-14 at 3 13 26 PM](https://user-images.githubusercontent.com/1813380/90289574-c9e9ba80-de41-11ea-873b-e510c60b5cc2.png)

## Related Tickets & Documents
https://github.com/orgs/forem/projects/6#card-43687049

## QA Instructions, Screenshots, Recordings
- pull down branch
- reboot app
- see new cron jobs scheduled, if you want, wait to see one run, like the one on the hour. 

![alt_text](https://media3.giphy.com/media/3o6Mb90J3mDKdAeSZO/giphy.gif)
